openslides/core/static/js/core/base.js

```diff
@@ -144,13 +145,22 @@ angular.module('OpenSlidesApp.core', [
             // set current language and return updated languages object array
             setCurrentLanguage: function (lang) {
                 var languages = this.getLanguages();
+                var plugins = OpenSlidesPlugins.getAll();
                 angular.forEach(languages, function (language) {
                     language.selected = false;
                     if (language.code == lang) {
                         language.selected = true;
                         gettextCatalog.setCurrentLanguage(lang);
                         if (lang != 'en') {
                             gettextCatalog.loadRemote("static/i18n/" + lang + ".json");
+                            // load language files from plugins
+                            if (plugins.length > 0) {
+                                angular.forEach(plugins, function (plugin) {
+                                    if ($.inArray(lang, plugin.languages) != -1) {
```

Comment:
Please use `plugins.indexOf(lang) != -1` instead of this jQuery method. (Or use a lodash method ...)